sfge/src/main/java/com/salesforce/config/UserFacingMessages.java

```diff
@@ -1,11 +1,32 @@
 package com.salesforce.config;
 
 /**
- * Contains error message constants that will be displayed to users. TODO: move all other
- * user-facing messages here.
+ * Contains error message constants that will be displayed to users.
+ * TODO: move all other user-facing messages here.
+ * TODO: Reorganize these messages into a layered system like in {@link com.salesforce.apex.jorje.ASTConstants}
  */
 public final class UserFacingMessages {
 
+    public static final class RuleDescriptions {
+        public static final String UNNECESSARILY_EXTENSIBLE_CLASS_RULE =
+                "Identifies classes that are extensible but never extended";
+        public static final String UNUSED_INTERFACE_RULE =
+                "Identifies interfaces that are declared but never implemented or extended";
```

Comment:
@teresa-allen-sfdc , these are the descriptions for the new rules.

Reply:
How about a little more information?
"Identifies non-global virtual classes that are never extended."
"Identifies non-global interfaces that are never implemented or extended."